bin/package-release

```diff
@@ -10,7 +10,21 @@
  * @license   http://www.opensource.org/licenses/mit-license.html MIT License
  */
 
-require_once __DIR__ . '/../vendor/autoload.php';
+$autoloadPaths = array(
+    // Try to load autoloader if this is the root project.
+    __DIR__ . '/../vendor/autoload.php',
+
+    // Try to load an autoloader if this is installed as a library for
+    // another root project.
+    __DIR__ . '/../../../autoload.php',
+);
+
+foreach ($autoloadPaths as $path) {
+    if (file_exists($path)) {
+        require_once $path;
+        break;
+    }
+}
```

Comment:
Good question. If neither is found the script will terminate shortly thereafter when there is an undefined class.